resources/views/components/front-block.blade.php

```diff
@@ -0,0 +1,28 @@
+<div {{ $attributes->merge(['class' => 'bg-white rounded p-4 mb-4']) }}>
+  <div class="flex flex-row mb-2">
+    <div class="flex-1 font-bold tracking-wide uppercase">{{ $title }}</div>
+    @if ($url)
+        <div class="rounded bg-gray-100 border px-2">
```

Comment:
Indentation varies between 2 and 4 spaces in this file.

Reply:
Fixed